src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/ConfiguredProjectImplicitActivationTracking.cs

```diff
@@ -1,147 +1,52 @@
 ﻿// Licensed to the .NET Foundation under one or more agreements. The .NET Foundation licenses this file to you under the MIT license. See the LICENSE.md file in the project root for more information.
 
-using System;
-using System.Collections.Generic;
 using System.ComponentModel.Composition;
-using System.Linq;
-using System.Threading;
-using System.Threading.Tasks;
-using System.Threading.Tasks.Dataflow;
-using Microsoft.VisualStudio.Threading.Tasks;
 
 namespace Microsoft.VisualStudio.ProjectSystem
 {
     /// <summary>
-    ///     Provides an implementation of <see cref="IConfiguredProjectActivationTracking"/> that is based on the results of <see cref="IActiveConfigurationGroupService"/>.
+    ///     Responsible for activating and deactiving <see cref="IImplicitlyActiveService"/> instances.
     /// </summary>
-    [Export(typeof(IConfiguredProjectImplicitActivationTracking))]
-    [AppliesTo(ProjectCapability.DotNet)]
-    internal class ConfiguredProjectImplicitActivationTracking : OnceInitializedOnceDisposed, IConfiguredProjectImplicitActivationTracking
+    [Export(ExportContractNames.Scopes.ConfiguredProject, typeof(IProjectDynamicLoadComponent))]
+    [AppliesTo(LoadCapabilities)]
+    internal partial class ConfiguredProjectImplicitActivationTracking : AbstractMultiLifetimeComponent<ConfiguredProjectImplicitActivationTracking.ConfiguredProjectImplicitActivationTrackingInstance>, IProjectDynamicLoadComponent
     {
+        // NOTE: Ideally this component would be marked with 'AlwaysApplicable' so that we always load
+        // IImplicitlyActiveService instances in all project types regardless of exported capabilities,
```

Comment:
It's not clear to me how this would be desirable since it's what we are trying to avoid doing. Is the concern about loading IImplicitlyActiveServices that we do not own?

Reply:
It would prevent the need to have LoadCapabilities match every single capability that every implementation of IImplicitlyActiveServices uses.